src/en/reference-bundle.md

```diff
@@ -0,0 +1,346 @@
+Title: Bundle reference
+
+# Bundle reference
+
+A bundle is defined by a [YAML-formatted][yaml] file with a fixed set of
+elements. This document will define the properties allowed in such a file and
+give the format of expressing them by way of examples.
+
+A bundle file makes available a number of charm series settings. A section on
+how a charm's series is ultimately determined is therefore presented first.
+
+Page [Charm bundles][charms-bundles] gives an overview of bundle usage.
+
+## Charm series
+
+What series a charm will use can be influenced in several ways. Some of these
+are set within the bundle file while some are not. When using bundles, the
+series is determined using rules of precedence (most preferred to least):
+
+ - the series stated in a charm URL (see `charm` under the
+   `<application name>` element)
+ - the series stated for an application (see `series` under the
+   `<application name>` element)
+ - the series given by the top level `series` element
+ - the top-most series specified in a charm's `metadata.yaml` file
```

Comment:
And the final default is the most recent, stable LTS

Reply:
Oh nice.